ImportExport/ObjectsToPython/ObjectsToPython.FCMacro

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+
+__Name__ = 'Objects To Python'
+__Comment__ = 'Exports objects from a FreeCAD project to a python script'
+__Author__ = 'Christi'
+__Version__ = '0.2'
+__Date__ = '2020-06-06'
+__License__ = 'LGPL-3.0-or-later'
+__Web__ = ''
+__Wiki__ = 'README.md'
+__Icon__ = ''
+__Help__ = ''
+__Status__ = ''
+__Requires__ = 'FreeCAD >= 0.17'
+__Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=22&t=34612'
+__Files__ = 'ObjectsToPython.py,Readme.md'
+
+VERSION_STRING = __Name__ + ' Macro v' + __Version__
+
+
+import FreeCAD
+import FreeCADGui
+import Part
+import re
+import sys
+from FreeCAD import Vector
+
+        
+        
+def exportObjectsToPython(doc = FreeCAD.ActiveDocument):
+    addScriptFunc("from FreeCAD import Vector, Placement, Rotation")
+    addScriptFunc("import Sketcher");
+    addScriptFunc("import Part");
+    addScript("")
+    objectlist = []
+    skipObjects = [('App::Line', 'X_Axis'), ('App::Line', 'Y_Axis'), ('App::Line', 'Z_Axis'),
+                   ('App::Plane', 'XY_Plane'), ('App::Plane', 'XZ_Plane'), ('App::Plane', 'YZ_Plane'),
+                   ('App::Origin', 'Origin')] 
+    
+    selection = FreeCADGui.Selection.getSelection() 
+    if not selection:
+        selection = doc.Objects
+    else:
+        expandSelection(selection)
+        
+    for obj in doc.Objects:  
+        objectlist.append(doc.getObject(obj.Name)) 
+         
+    for obj in selection:    
+        if obj.TypeId == "Sketcher::SketchObject": 
+            addScriptFunc("")
+            addScriptFunc("def createSketch_%s(doc):"%varname(obj))
+            addObject(doc, obj, objectlist)
+            addScript("return " + varname(obj))
+        
+    addScriptFunc("")
+    addScriptFunc("")
+    addScriptFunc("def make_%s():"%doc.Name)
+    if not selection:
+        addScript("doc = App.newDocument('%s')"%doc.Name)
+    else:
+        addScript("doc = App.ActiveDocument")
```

Comment:
`App` was not in the import statements of the generated script. Also, `app.activeDocument()` is safer because `app.ActiveDocument` could be outdated.